lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -90,10 +90,16 @@ function computeDescription(ast, message) {
 
 /**
  * Collapses a jsdoc comment into a single line and trims whitespace.
- * @param {string=} comment
+ * @param {import('typescript').JSDoc['comment']} comment
```

Comment:
tsc 4.3 added [support for `@link` tags](https://github.com/microsoft/TypeScript/pull/41877) in 4.3, so jsdoc comments aren't always just simple strings anymore, sometimes they're structured data. Our `UIStrings` jsdoc comments are still just strings for now, though, so everything still works the same.

There is some movement in their code to take a more array-of-text-nodes generalized approach to jsdoc comments, so I left a comment in here in case it ever does start failing.